src/main/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceService.java

```diff
@@ -39,6 +40,23 @@ public ObjectResponse create(String accountUrn, ObjectCreate createObject) {
     }
 
     @Override
+    public Optional<ObjectResponse> update(String accountUrn, ObjectResponse updateObject) {
```

Comment:
If update does not consume ObjectResponse, the find functions also should not return it. Rather all DAO functions should use ObjectEntity, it is the task of the RDAO to convert the REST objects to DAO objects. The DAO is responsible to load and store these only.